owners/OWNERS.example.yaml

```diff
@@ -54,21 +60,49 @@
 # appropriate OWNERS file should be created/updated in that subdirectory.
 - this/pattern/will/be/ignored.txt: doesntmatter
 
-# Filename and pattern rules may be combined with brace expansion.
+# Filename and pattern rules may be combined with brace expansion. The pattern
+# parser uses `[minimatch](https://www.npmjs.com/package/minimatch)`, the
+# matching library used internally by npm.
 - "*.{js,css}": frontend
 - "package.{,lock.}json": somebodyelse
 
+# Prepending a user or team name with a `?` will prevent this owner from being
+# automatically notified or requested as a reviewer. They may still be requested
+# or tagged manually.
+- "**/*.css":
+  - "?someoneimportant"
+  - "?ampproject/important-team"
+
+# Prepending a user or team name with a `?` will prevent this owner from being
+# automatically notified or requested as a reviewer. They are still an owner,
+# may still be requested or tagged manually.
+- "**/*.css": "?someoneimportant"
+- "yarn.lock": "?ampproject/important-team"
+
+# Alternately, the `#` modifier will ensure that the user is @mentioned in any
+# PR touching those files, even if they are not added as a reviewer. The bot
+# will add a comment to these PRs to notify the owners.
+- "**/*.test.js": "#stakeholder"
```

Comment:
The line of thought was:
- a lot of existing owners files had `@` scattered about, because at the time people weren't sure if it was needed or not (just today I reviewed a PR where someone prefixed a team with `@` in an owners file out of habit)
- to reduce clutter, the parser was designed to not need it, and to ignore it if present
- the thought was `#` would be familiar enough as a tagging symbol
- `!` was considered but then planned to be used for "required" reviewers

I think the right move will be to drop the special syntax entirely and opt instead for something more like `notify: always` or something similar

Reply:
@mrjoro Would love your thoughts & opinions in https://github.com/ampproject/amp-github-apps/issues/442

Reply:
OK, any of these formats sound fine to me, including using text.  As long as its documented. :)

I commented on that issue as well.